addOns/selenium/src/main/zapHomeFiles/scripts/templates/selenium/Selenium default template.js

```diff
@@ -0,0 +1,13 @@
+// The browserLaunched function is called whenever a browser is launched from ZAP using selenium.
+function browserLaunched(wd, requester, providedBrowserId, proxyAddress, proxyPort) {
+	var url = wd.getCurrentUrl();
+	while (! url.startsWith("http")) {
```

Comment:
- We should provide this behaviour with the parameter/utils object (e.g. `utils.waitFor('http://example.org')` or `utils.waitForPageAccess()`), this is expected to be used by most scripts.
 - This needs a timeout, otherwise it might lead to an infinite loop (cases in the following comment).
 - It needs to obtain the current URL inside the loop otherwise this will not work properly if the browser is started with `about:blank` or if the browser is closed before accessing any URL.